src/agent/coverage/fuzz/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "coverage-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+tempfile = "3.7"
+debuggable-module = { path = "../../debuggable-module" }
+
+
+[dependencies.coverage]
+path = ".."
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
+
+[profile.release]
+debug = 1
+
+[[bin]]
+name = "fuzz_target_1"
```

Comment:
I think you have to change the name and target values to match your actual filename.

Reply:
Good catch!